azure/template.json

```diff
@@ -11,6 +11,10 @@
         "serviceName": {
             "type": "string"
         },
+        "certificateName": {
```

Comment:
Can this and its references be renamed to keyvaultCertificateName

Reply:
@adam230594  corrected it